test/test_jit.py

```diff
@@ -1236,124 +1236,164 @@ def observer(self, x, name):
         self.assertTrue(trace.conv1.weight in list(value_stats.values()))
         self.assertTrue(trace.conv1.bias in list(value_stats.values()))
 
-    def test_insert_quantdequant_consecutive_qnodes_script(self):
-        class testModule(torch.jit.ScriptModule):
-            def __init__(self):
-                super(testModule, self).__init__()
-                self.conv1 = nn.Conv2d(1, 20, 5, 1)
-
-            @torch.jit.script_method
-            def forward(self, x):
-                x = F.relu(self.conv1(x))
-                return x
+    def test_insert_quantdequant(self):
+        # Test Class to generate default qparams
+        script_only = True
+        x1 = torch.ones([1, 1, 1, 1])
 
-        trace = testModule()
+        class TestGenQParam:
```

Comment:
This test is huge, and it's very hard to understand what it checks. Please replace it with a small unit test that checks **only** what the pass is supposed to do.